frontend/src/components/pages/topics/TopicConfiguration.tsx

```diff
@@ -65,47 +65,49 @@ export default class ConfigurationEditor extends Component<{
             content: <Observer>{() => {
                 const isCustom = this.modalValueType == 'custom';
 
-                return <div>
-                    <p>Edit <code>{configEntry.name}</code> configuration for topic <code>{this.props.targetTopic}</code>.</p>
-                    <div style={{
-                        padding: '1em',
-                        background: 'rgb(238, 238, 238)',
-                        color: 'hsl(0deg 0% 50%)',
-                        borderRadius: '8px',
-                        margin: '1em 0'
-                    }}>{configEntry.documentation}</div>
-
-                    <div style={{ fontWeight: 'bold', marginBottom: '0.5em' }}>Value</div>
-                    <Radio.Group className="valueRadioGroup" value={this.modalValueType} onChange={e => this.modalValueType = e.target.value} >
-                        <Radio value="default">
-                            <span>Default: </span>
-                            <span style={{ fontWeight: 'bold' }}>{friendlyDefault}</span>
-                            <div className="subText">Inherited from {defaultSource}</div>
-                        </Radio>
-                        <Radio value="custom">
-                            <span>Custom</span>
-                            <div className="subText">Set at topic configuration</div>
-                            <div onClick={e => {
-                                if (isCustom) {
-                                    // If the editor is *already* active, we don't want to propagate clicks out to the radio buttons
-                                    // otherwise they will steal focus, closing any select/dropdowns
-                                    e.stopPropagation();
-                                    e.preventDefault();
-                                }
-                            }}>
-                                <ConfigEntryEditor className={'configEntryEditor ' + (isCustom ? '' : 'disabled')} entry={configEntry} />
-                            </div>
-                        </Radio>
-                    </Radio.Group>
-
-                    {this.modalError && <Alert status="error" style={{ margin: '1em 0' }}>
-                        <AlertIcon />
-                        {this.modalError}
-                    </Alert>}
-
-                </div>
+                return(
+                    <ChakraProvider theme={redpandaTheme} disableGlobalStyle={true} disableEnvironment={true}>
```

Comment:
Nice one! I think as long as we add a follow up GH issue to remove the theme after using the chakra modal, this will work fine in the meantime